GoogleSignIn/Sources/GIDSignInPreferences.m

```diff
@@ -94,6 +100,18 @@ + (NSString *)googleUserInfoServer {
   return kUserInfoServer;
 }
 
++ (NSURL *)authorizationEndpointURL {
+  NSString *authorizationEnpointURL = [NSString stringWithFormat:kAuthorizationURLTemplate,
+      [GIDSignInPreferences googleAuthorizationServer]];
```

Comment:
You can do `[self googleAuthorizationServer]` instead, which is slightly preferable as it doesn't require an update if the class ever gets renamed.